### PR TITLE
Allow numbers as modifiers

### PR DESCRIPTION
**Pull request by geroldboehler**
Hi, please consider my PR. We have the need to use numbers as modifiers and i think maybe others might so too.

### CODE REVIEW

**Comment by patric-eberle** [comment]
@c01nd01r could you please review this change request? We would really like to be able to use numbers since we use a bootstrap based variant system, which works with numbers.

File: src/bem-cn/bem-names.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default function bemNames(entitys, delimiters) {
       /* eslint-disable no-param-reassign */
       if (val === true) {
         prev += ' ' + resultString + delims.mod + name;
-      } else if (isString(val)) {
+      } else if (isString(val) || typeof(val) === 'number') {
```
Comment (c01nd01r) [inline src/bem-cn/bem-names.js]: Can you replace `number` detection with `isNumber` helper in [/src/utils.js](https://github.com/antiloopgmbh/vue-bem-cn/blob/master/src/utils.js) something like this:
```
const isNumber = val => typeof val === 'number' && isFinite(val);

**Comment by geroldboehler** [comment]
@c01nd01r Updated as requested

**Comment by c01nd01r** [comment]
Hey, guys, sorry for the long wait.
Update was sent to npm.

**Pull request merged**